### PR TITLE
Fix build errors and finish ShooterSubsystem

### PR DESCRIPTION
**Pull request by jamietheraccoon**
Project now builds

Added PID controllers to the ShooterSubsystem and inverted one of the motors to clean up the code

### CODE REVIEW

File: src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsytem.java [hunk 1]
```diff
@@ -8,7 +8,9 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import io.github.oblarg.oblog.Loggable;
 
-public class DrivebaseSubsystem extends SubsystemBase implements Loggable{
+import frc.team2412.robot.commands.DriveCommand;
+
+public class DrivebaseSubsytem extends SubsystemBase implements Loggable{
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsytem.java]: Please rename the file & class to fix the typo: DrivebaseSubsytem  ->  DrivebaseSubsystem

File: src/main/java/frc/team2412/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -48,14 +56,25 @@
...
 	// Create the example subsystem
-	public ShooterSubsystem(CANSparkMax motor1, CANSparkMax motor2, CANSparkMax motor3, Servo servo1, Servo servo2) {
-		this.motor1 = motor1;
-        this.motor2 = motor2;
+	public ShooterSubsystem(CANSparkMax leftMotor, CANSparkMax rightMotor, CANSparkMax motor3, Servo servo1, Servo servo2) {
+        this.leftMotor = leftMotor;
+        this.leftPIDController = this.leftMotor.getPIDController();
+
+        this.rightMotor = rightMotor;
+        this.rightMotor.setInverted(true);
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/subsystems/ShooterSubsystem.java]: Best to also set the motors to coast mode, else they're likely going to be damaged or draw a lot of current when they are set to stop after running at high speed.

**Review by jamesdooley4 — CHANGES_REQUESTED** [review]
Please fix the typo in the drive base subsystem name

**Comment by jamietheraccoon** [comment]
Implemented suggested changes

**Comment by jbko6** [comment]
good job

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**